### PR TITLE
DOC: create shared includes for comparison docs, take III

### PR DESCRIPTION
**Pull request by afeld**
From original pull request (https://github.com/pandas-dev/pandas/pull/38771):

> This will help ensure consistency between the examples.

- [ ] ~~closes #xxxx~~
- [x] tests added / passed
- [ ] ~~passes `black pandas`~~
- [ ] ~~passes `git diff upstream/master -u -- "*.py" | flake8 --diff`~~
- [ ] ~~whatsnew entry~~

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -49,7 +49,10 @@ ignore = E203,  # space before : (needed for how black formats slicing)
          E711,  # comparison to none should be 'if cond is none:'
 
 exclude =
-    doc/source/development/contributing_docstring.rst
+    doc/source/development/contributing_docstring.rst,
+    # work around issue of undefined variable warnings
+    # https://github.com/pandas-dev/pandas/pull/38837#issuecomment-752884156
+    doc/source/getting_started/comparison/includes/*.rst
```
Comment (afeld) [inline setup.cfg:55]: Per suggestion in https://github.com/pandas-dev/pandas/pull/38837#issuecomment-752884156.

File: doc/source/getting_started/comparison/includes/if_then.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+The same operation in pandas can be accomplished using
```
Comment (jreback) [inline doc/source/getting_started/comparison/includes/if_then.rst:1]: are any of these shared across compraisons? its not big deal, but you can simply re-use them.

**Review by jreback — CHANGES_REQUESTED** [review]

File: doc/source/getting_started/comparison/includes/if_then.rst (hunk 2 above)
Comment (afeld) [reply doc/source/getting_started/comparison/includes/if_then.rst:1]: Sorry, what do you mean by "these"?

Comment (jreback) [reply doc/source/getting_started/comparison/includes/if_then.rst:1]: oh it looks like you are doing what i thought (e.g. putting includes in a common location) and then using for comparison_excel/sas/r etc. which is great.

so nvm

**Comment by afeld** [comment]
Tests pass! 🎉

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @afeld 

pls have a look at the built docs and issue a followup PR if needed.

OT: pls comment on your open PRs / merge master for them to see what we need to merge (IIRC some were WIP)

**Comment by afeld** [comment]
Yep yep, they look how I'd expect 👍 

- https://pandas.pydata.org/docs/dev/getting_started/comparison/comparison_with_sas.html
- https://pandas.pydata.org/docs/dev/getting_started/comparison/comparison_with_stata.html
- https://pandas.pydata.org/docs/dev/getting_started/comparison/comparison_with_sql.html

**Pull request merged**